pytest_pootle/fixtures/views.py

```diff
@@ -132,11 +135,21 @@ def project_views(request, client):
 
 
 @pytest.fixture(params=TP_VIEW_TESTS.keys())
-def tp_views(request, client):
+def tp_view_test_names(request):
+    return request.param
+
+
+@pytest.fixture(params=(None, 'member', 'admin'))
```

Comment:
Thinking aloud: isn't this a use case for [`@pytest.mark.parametrize`](http://pytest.org/latest/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions) in the `tp_views` test function?